packages/Python/lldbsuite/test/test_result.py

```diff
@@ -109,6 +108,14 @@ def getCategoriesForTest(self, test):
         """
         Gets all the categories for the currently running test method in test case
         """
+
+        # It isn't possible to reliably attach categories to inline tests
+        # because they all share the same instance method: punching in the
+        # category there causes the wrong set of tests to be included in it.
```

Comment:
I'd rephrase this, maybe something like "punching in the category for one inline test causes all inline tests to belong to that category".